.buildbot.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
If we *really* to use bash (do we?) let's at least use `/usr/bin/env bash`.

Reply:
Fixed in 7889640.

Reply:
Oh well. Let's use `/usr/bin/env bash` at least.